modules/op-release-notes-1-20-0.adoc

```diff
@@ -0,0 +1,289 @@
+
+* Before this update, the {OCP} Console stated a 'Cancelling' state for `PipelineRun` objects with failed `finally` tasks, even when the actual status of the `PipelineRun` was `Canceled`, causing confusion. With this update, the issue is fixed.
+
+* Before this update, the `PipelineRun` Events tab did not show events, due to an issue with the Pipeline Console plugin. As a consequence, you could not monitor the events in real-time. With this update, the issue is fixed.
+
+* Before this update, the pipeline builder fetched only a limited number of tasks from {artifact-hub}, resulting in some tasks not being available. This resulted in an incomplete pipeline creation. With this update, the interface fetches the complete list of tasks from {artifact-hub} on search, showing all available tasks in the pipeline builder.
+
+* Before this update, running `fbc-fips-check-oci-ta` tasks during git cloning caused temporary resource unavailability, stalling the git resolver and preventing the resolution of `ResolutionRequests`. This resulted in a build failure with the following error message: `cannot fork() for remote-https: Resource temporarily unavailable`. With this update, the issue is fixed.
+
+* Before this update, `TaskRun` and `CustomRun` creation could fail immediately on transient mutating-webhook timeouts, causing flakiness on busy clusters. With this update, the system  applies exponential backoff, configurable via the `wait-exponential-backoff`  config map and controlled by the `enable-wait-exponential-backoff` setting:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: feature-flags
+  namespace: tekton-pipelines
+data:
+  enable-wait-exponential-backoff: "true"
+#...
+----
+
+* Before this update, the controllers used a fixed thread count, limiting concurrency. With this update, you can override `threads-per-controller` by setting the `THREADS_PER_CONTROLLER` environment variable, allowing finer control over controller concurrency.
+
+* Before this update, `podTemplate` fields in `TaskRunSpec` CR did not support parameter substitution, limiting matrix and multi-arch patterns. With this update, the controller substitutes parameters in `podTemplate` fields for all `TaskRuns` and `TaskRunSpecs`.
+
+* Before this update, the `onError` block in `pipeline` v1beta1 did not support variables. With this update, the controller  resolves the `onError` variables, improving error handling flexibility.
+
+* Before this update, git resolver shell-outs sometimes failed to inherit environment variables, breaking environment-driven authentication or configuration. With this update, the resolver passes the pod environment correctly to all git subprocesses.
+
+* Before this update, git resolver deployments could leave zombie git processes running, consuming resources. With this update, resolvers  use a `tini`-based image and entrypoint to actively clean up subprocesses.
```

Comment:
@athorp96 isn't `tini` based only upstream ?

Reply:
`tini` is [used downstream](https://github.com/openshift-pipelines/tektoncd-pipeline/blob/60a1d516229066f70233c56392a66cbc6a16e618/.konflux/dockerfiles/resolvers.Dockerfile#L54) as well, but it's not a `tini`-based image. It's not new to 1.20 though. Both of these changes were introduced in 1.19: [pod environment in git](https://github.com/openshift-pipelines/tektoncd-pipeline/blob/0a7000aba8ab139c014af62b191890818ca9ab82/upstream/pkg/resolution/resolver/git/repository.go#L122) and [tini entrypoint](https://github.com/openshift-pipelines/tektoncd-pipeline/blob/0a7000aba8ab139c014af62b191890818ca9ab82/.konflux/dockerfiles/resolvers.Dockerfile#L53)